packages/asset-service/src/utils.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  CHAIN_NAMESPACE,
+  CHAIN_REFERENCE,
+  ChainId,
+  ChainNamespace,
+  ChainReference,
+  fromChainId
+} from '@shapeshiftoss/caip'
+
+// https://www.coingecko.com/en/api/documentation - See asset_platforms
+export const chainPartsToCoingeckoAssetPlatform = (
+  chainNamespace: ChainNamespace,
+  chainReference?: ChainReference
+): string => {
+  return (() => {
+    switch (chainNamespace) {
+      case CHAIN_NAMESPACE.Ethereum:
+        return 'ethereum'
+      case CHAIN_NAMESPACE.Cosmos:
+        return chainReference === CHAIN_REFERENCE.CosmosHubMainnet ? 'cosmos' : 'osmosis'
+      default:
```

Comment:
Either type the return value as `string | null` or remove the default case. Another solution might be to use `Exclude<>` to restrict `CHAIN_NAMESPACE.Bitcoin` at type level.

If we keep it, can we add a comment explaining that the reason the switch case doesn't include `CHAIN_NAMESPACE.Bitcoin` is because Bitcoin isn't present in https://api.coingecko.com/api/v3/asset_platforms ? 🙏🏽